backend/App/Program.cs

```diff
@@ -10,6 +10,10 @@
 builder.Services.AddDbContext<DataContext>(
     opt => opt.UseNpgsql(builder.Configuration.GetConnectionString("Database")));
 
+builder.Services.AddCors(opt =>
+    opt.AddPolicy("CorsPolicy", policy => 
+        policy.AllowAnyHeader().AllowAnyMethod().WithOrigins("http://localhost:3000")));

```

Comment:
question: Should origin string be hardcoded here? Wouldn't we rather want to store it in some env/secret to be able to properly set it on production environment?